### PR TITLE
Bugfix for query/response based simulator

### PR DESCRIPTION
**Pull request by nagkumar91**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_simulator.py [hunk 1]
```diff
@@ -398,14 +402,16 @@ async def _generate_query_responses(
...
         try:
             query_responses = query_flow(text=text, num_queries=num_queries)
+            print(type(query_responses))
+            print(query_responses)
             if isinstance(query_responses, dict):
                 keys = list(query_responses.keys())
                 return query_responses[keys[0]]
             return json.loads(query_responses)
         except Exception as e:
+            import pdb; pdb.set_trace()
```
Comment (MilesHolland) [inline sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_simulator.py:414]: Pdb statement

Comment (nagkumar91) [reply sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_simulator.py]: Good catch! Fixed it

**Pull request merged**